frontend/src/components/Utils/time.js

```diff
@@ -93,15 +97,28 @@ export function getISODate(msTimestamp, timezone=null) {
 }
 
 /**
- * Get a time in 24-hour ('HH:mm') form. 
+ * Get a time in 24-hour ('HH:mm') format. 
  * 
  * @param {number} msTimestamp Timestamp, in milliseconds since epoch.
  * @param {string} timezone The timezone which the string should be returned in.
- * @return {string} The time in 24-hour (HH:mm) form.   
+ * @return {string} The time in 24-hour (HH:mm) format.   
  */
 export function get24hTime(msTimestamp, timezone=null) {
```

Comment:
```suggestion
export function get24hTime(msTimestamp, timezone = null) {
```

Also add space around "=" for the rest of the files in the PR